necst/rx/spectrometer.py

```diff
@@ -100,6 +98,15 @@ def __init__(self) -> None:
         self.logger = self.get_logger()
 
         self.resizers = defaultdict(lambda: Resize(1))
+
+        try:
+            from neclib.devices import Spectrometer
+        except ImportError:
+            self.logger.error(
+                "Spectrometer not found; no connection or no configuration"
```

Comment:
同じような内容を繰り返しているので、以下のようなメッセージにしてほしいです。

```suggestion
                "Configuration for spectrometer not found; no spectral data would be recorded"
```